lib/volcanic/authenticator/v1/identity.rb

```diff
@@ -96,6 +96,16 @@ def token(audience: [], exp: nil, nbf: nil, single_use: false)
         request_token '/token/generate', payload
       end
 
+      def activate!
```

Comment:
is there a Jira that is linked to this TODO? if so, can you add it to the comment so that it is easy to find later?